SemanticMediaWiki.php

```diff
@@ -57,6 +57,11 @@
 		3009 => false,
 		3012 => true,
 		3013 => false,
+		3018 => true,
```

Comment:
Talk namespaces also should have been added here with => false.

Reply:
I opened a new PR implementing those changes: https://github.com/miraheze/mw-config/pull/5065